hyperspy_gui_traitsui/__init__.py

```diff
@@ -16,7 +16,6 @@
 # You should have received a copy of the GNU General Public License
 # along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
 
-from hyperspy_gui_traitsui.version import __version__
```

Comment:
Why removing this?

Reply:
Good question... this is clearly a mistake!